common/lib/xmodule/xmodule/tests/test_course_module.py

```diff
@@ -277,3 +277,60 @@ class DiscussionTopicsTestCase(unittest.TestCase):
     def test_default_discussion_topics(self):
         d = get_dummy_course('2012-12-02T12:00')
         self.assertEqual({'General': {'id': 'i4x-test_org-test_course-course-test'}}, d.discussion_topics)
+
+
+class TeamsConfigurationTestCase(unittest.TestCase):
+    """
+    Tests for the configuration of teams and the helper methods for accessing them.
+    """
+    counter = 0
+
+    def setUp(self):
+        super(TeamsConfigurationTestCase, self).setUp()
+        self.course = get_dummy_course('2012-12-02T12:00')
+
+    def add_team_configuration(self, max_team_size=3, topics=None):
+        """ Add a team configuration to the course. """
+        if topics is None:
+            topics = []
+        self.course.teams_configuration = {"max_team_size": max_team_size, "topics": topics}
+
+    def make_topic(self):
+        """ Make a sample topic dictionary. """
+        topic_id = "topic_id_{}".format(self.counter)
+        self.counter = self.counter + 1
+        return {"display_name": "Display Name", "description": "Description", "id": topic_id}
+
+    def test_teams_enabled_no_teams(self):
+        # Make sure we can detect when no teams exist.
+        self.assertFalse(self.course.teams_enabled)
```

Comment:
This test seems to me like it's more of a test of the fact that new courses have blank `teams_configuration`s by default (a useful test case), but if that's what you're testing it would be a bit more robust to instantiate the course you're testing in this method, rather than rely on the `setUp` code.

Another interesting test case would be removing topics from a team configuration and asserting then that teams_enabled is False. You could probably implement both of those cases in this one meth